src/python-common/ceph/deployment/service_spec.py

```diff
@@ -679,10 +679,10 @@ def __init__(self,
             config=config, networks=networks)
 
         #: RADOS pool where NFS client recovery data is stored.
-        self.pool = pool
+        self.pool = pool or 'nfs-ganesha'
```

Comment:
Define a constant for default pool name. Then we can remove from nfs module https://github.com/ceph/ceph/blob/master/src/pybind/mgr/nfs/utils.py#L3